modules/persistent-storage-local-discovery.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent-storage-local.adoc
+
+[id="local-storage-discovery_{context}"]
+= Automating discovery and provisioning for local storage devices
+
+The Local Storage Operator automates local storage discovery and provisioning. With this feature, you can simplify installation when dynamic provisioning is not otherwise available during deployment, such as with bare metal, VMware, or AWS store instances with attached devices.
+
+Use the following procedure to automatically discover local devices, and to automatically provision local volumes for selected devices.
+
+.Prerequisites
+
+* The Local Storage Operator is installed.
+
+* Local disks are attached to {product-title} nodes.
+
+* You have access to the {product-title} web console and the `oc` command-line interface (CLI).
+
+.Procedure
+. To display a continuous list of available devices on a node:
+
+.. Log in to the {product-title} web console.
+
+.. Navigate to *Compute* -> *Nodes*.
+
+.. Click the node name you want to open. The "Node Details" page is displayed.
+
+.. From the *Actions* drop-down list, select *Discover nodes* to allow automatic discovery of the selected  devices on the node.
+
+.. Select the *Disks* tab to display the list of the selected devices.
++
```

Comment:
I am not sure if this how we want to drive the UI but I already noticed a bug. When I created a local-volume discovery object and then deleted it - I don't see "DIscover Nodes" Action in the dropdown anymore. I am not sure if it was there to begin with because I created discovery from the YAML and not from the console.

But once discovery was started, it does show discovered results.

Reply:
I was never able to find the "Discover Nodes" option in the Actions drop-down, although device discovery was started. Should this be reported as a bug? I'm leaving this list of steps about "Discover nodes" in for now, but if we can't validate, then it will have to come out.

Reply:
Yes I dont see Discover Nodes option either. We should remove it and ask users to explicitly create discovery request from LSO UI.

Reply:
@gnufied - I removed the line about "Discover Nodes." I also added a task for creating the local discovery volume request from the LSO UI, and moved that to be the first task in the procedure. Do you think this ordering makes sense?